### DISCUSSION TITLE
How to display categories and tags in the sidebar ?

### DISCUSSION THREAD

**Discussion by ghost** [mmistakes/minimal-mistakes#2561] [Q&A] [answered]
The question is in the title.
- display categories and number, click to display all posts of the category
- display tags and number, click to display all posts with this tag

**Comment by mmistakes** [comment]
Not supported by theme. You’ll need to customize layouts to do this.

**Comment by ghost** [comment]
The _includes/sidebar.html  layout ?
Do you have references to help me ?

rgds,

**Comment by mmistakes** [comment, marked as answer]
https://superdevresources.com/tag-cloud-jekyll/
http://tomnorian.com/a-liquid-tag-cloud-for-jekyll-blogs.html

There are more resources like these. Search around for Jekyll tag clouds. Adapt their code and add to MM layouts or sidebar as you need.

**Comment by ghost** [comment]
Thank you so much !
Excellent theme I discovered day after day.
